linux_os/guide/system/permissions/partitions/mount_option_boot_nosuid/rule.yml

```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Add nosuid Option to /boot'
+
+description: |-
+    The <tt>nosuid</tt> mount option can be used to prevent
+    execution of setuid programs in <tt>/boot</tt>. The SUID and SGID permissions
+    should not be required in these user data directories.
+    {{{ describe_mount(option="nosuid", part="/boot") }}}
+
+rationale: |-
+    The presence of SUID and SGID executables should be tightly controlled. Users
+    should not be able to execute SUID or SGID binaries from boot partitions.
+
+severity: unknown
+
+identifiers:
+    cce@rhel8: 81033-3
+
+references:
```

Comment:
Can likely nab the NIST references from other partition option checks.

Reply:
So `nist: CM-7,MP-2` and `nist-csf: PR.IP-1,PR.PT-2,PR.PT-3`?